ui/pages/settings/contact-list-tab/add-contact/add-contact.test.js

```diff
@@ -64,9 +64,10 @@ describe('AddContact component', () => {
 
     const input = getByTestId('ens-input');
     fireEvent.change(input, { target: { value: 'invalid address' } });
-    setTimeout(() => {
-      expect(getByText('Recipient address is invalid')).toBeInTheDocument();
-    }, 600);
+
+    await waitFor(() =>
+      expect(getByText('Recipient address is invalid')).toBeInTheDocument(),
+    );
```

Comment:
You can [findBy*](https://testing-library.com/docs/dom-testing-library/api-async#findby-queries) queries here and avoid using the `waitFor`.